### ISSUE TITLE
Snake move around weirdly and quickly get "sleepy"

### ISSUE THREAD

**Issue by akien-mga** [KOBUGE-Games/jetpaca#16]
The snakes (visible in particularly in the forest_fun level) have a weird way to move around, and they quickly end up kind of sleepy or deactivated.

We need to investigate if it's a regression (could be that they used to behave better but something broke in their logic in newer Godot version), or if they were always like that. In both case we should try to fix it.

**Comment by bojidar-bg** [comment]
I think they don't become "sleepy", but they just try to eat their tails or the terrain, and end up unable to move. (but I might be wrong, since I haven't tested in a long time)

**Comment by akien-mga** [comment]
Yeah it's mostly that they are dumb around objects and have a clumsy way to move.
![spectacle je6586](https://cloud.githubusercontent.com/assets/4701338/16931082/63c6fe30-4d3f-11e6-95e1-fa2d128dbb53.png)
We should just aim a improving their AI a bit to make them feel a bit more natural and challenging, and ensure they gut stuck less easily.
